src/Symfony/Component/Intl/Resources/bin/transliterator-emoji-rules.php

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$sourceFilename = __DIR__.'/emoji-test.txt';
+
+Builder::downloadSource($sourceFilename);
+$rules = Builder::buildRules($sourceFilename);
+Builder::saveRules($rules);
+
+class Builder
+{
+    public static function downloadSource(string $sourceFilename): void
+    {
+        if (!file_exists($sourceFilename)) {
+            copy('https://www.unicode.org/Public/emoji/15.0/emoji-test.txt', $sourceFilename);
```

Comment:
This file is not supposed to hold the reference "translation" of an emoji. It's a complete list for display testing and keyboard implementation. There is no guaranty about the format being `Format: code points; status # emoji name` forever and it only supports English.

You should use CLDR data, like I do in emoji-search: https://github.com/jolicode/emoji-search/blob/master/tools/build-released.php - it's available as JSON and XML too.

Annotations provide names and keywords for Unicode characters.

What you want here is the "[CLDR Short Name](https://unicode.org/emoji/format.html#col-name)", as shown in bold here: https://unicode-org.github.io/cldr-staging/charts/latest/annotations/romance.html

Reply:
We talk IRL with @damienalexandre, and we decided: 

* to use https://github.com/unicode-org/cldr/blob/main/common/annotations/fr.xml
* and https://github.com/unicode-org/cldr/blob/main/common/annotationsDerived/fr.xml
* using the `tts` entries (text to speach)
* and to build one file per local

Thanks

Reply:
Implemented 😍